website/docs/r/metadata_source.html.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: "sumologic"
+page_title: "SumoLogic: sumologic_metadata_source"
+description: |-
+  Provides a Sumologic Metadata (Tag) source. This source allows you to collect tags from EC2 instances running on AWS.
+---
+
+# sumologic_polling_source
```

Comment:
Should this be metadata?

Reply:
The header

Reply:
not sure if this was discussed earlier, did we decide on adding the metadata source as a separate source resource instead of to the polling source like what we did for cloudwatch?

Reply:
Yes, as per the earlier discussion it was decided to add separate resources for all AWS sources so that we have 1:1 mapping with what we show on UI. 

Also, as mentioned in the description
>Have created a new resource because the tag_filters for this source type accepts a list of strings as compared to tag_filters of polling source which is a list of maps with type, namespace and tags as keys.

Reply:
I am working on splitting the `polling_source` into separate sources and will create a new PR for that.

Reply:
thanks for the clarification